06-classification_continued.Rmd

```diff
@@ -479,7 +576,8 @@ new data: if we had a different training set, the predictions would be
 completely different.  In general, if the model *is influenced too much* by the
 training data, it is said to **overfit** the data.
 
-You can see this effect in the plots below as we vary the number of neighbours $K$ in (1, 7, 20, 200):
+You can see these two effects in Figure \@ref(fig:06-decision-grid-K),
+which shows how the classifier changes as we set the number of neighbours $K$ to 1, 7, 20, and 200.
 
 <center>
 ```{r 06-decision-grid-K, echo = FALSE, fig.height = 7, fig.width = 10, fig.cap = "Effect of K in overfitting and underfitting"}
```

Comment:
Can we show an even more extreme K, say 300? Where it gets as bad as it can? I think that might help get the point across better.

Reply:
done